app/controllers/users_controller.rb

```diff
@@ -33,6 +49,20 @@ def accept_screen(method, path)
     end
   end
 
+  def next_state
```

Comment:
We can't use `aasm` on Publish and TBH a simple approach should suffice?

Reply:
I think we can [define `aasm_write_state`](https://github.com/aasm/aasm/blob/master/lib/aasm/persistence/plain_persistence.rb#L14) to show AASM how to persist the state value.

Reply:
(but, again, we don't _have_ to use aasm, but it might be tidier than doing it this way ... worth investigating)